cmd/juju/application/deployer/bundlehandler_test.go

```diff
@@ -168,22 +168,6 @@ func (s *BundleDeployRepositorySuite) TestDeployBundleWithInvalidSeries(c *gc.C)
 	s.expectWatchAll()
 	s.expectResolveCharm(nil)
 
-	s.expectAddMachine("0", "bionic")
-	s.expectResolveCharm(nil)
-	s.expectAddCharm(false)
-	wordpressCurl := charm.MustParseURL("cs:wordpress-47")
-	charmInfo2 := &apicharms.CharmInfo{
-		Revision: wordpressCurl.Revision,
-		URL:      wordpressCurl.String(),
-		Meta: &charm.Meta{
-			Series: []string{"xenial", "bionic"},
-		},
-	}
-	s.expectCharmInfo(wordpressCurl.String(), charmInfo2)
-	s.expectDeploy()
-
-	s.expectAddMachine("0", "focal")
-	s.expectAddCharm(false)
```

Comment:
Is this deleted because of the change in order of processing now?

Reply:
yep